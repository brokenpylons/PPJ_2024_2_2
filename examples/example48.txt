(4.6 * x^2 - 3.1 * y) / (1.2^2) + e^(0.5 * z) / (2.3 * w)
