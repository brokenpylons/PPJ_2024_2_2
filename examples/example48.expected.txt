lparen("(") real("4.6") times("*") variable("x") pow("^") real("2") minus("-") real("3.1") times("*") variable("y") rparen(")") divides("/") lparen("(") real("1.2") pow("^") real("2") rparen(")") plus("+") variable("e") pow("^") lparen("(") real("0.5") times("*") variable("z") rparen(")") divides("/") lparen("(") real("2.3") times("*") variable("w") rparen(")") 
